web/session.go

```diff
@@ -125,6 +125,35 @@ func loginOK(r *rest.Request) bool {
 	return true
 }
 
+func loginWithTokenOK(r *rest.Request, token string) bool {
+	restToken, err := auth.ParseRestToken(token)
+	//validToken, err := auth.ValidateRestToken(r.Request, token)
+	if err != nil {
+		msg := "Unable to parse rest token"
+		plog.WithError(err).Info(msg)
+		return false
+	} else if !restToken.HasAdminAccess() {
```

Comment:
For this PR, yes. For the CC zenpack to connect we might need to whitelist an endpoint or two? Unless that always goes out through the host network stack rather than through our proxy.